### PR TITLE
Adding --rbb-port=PORT parameter to the Emulator

### PR DESCRIPTION
**Pull request by noureddine-as**
This is added to allow defining a port for the JTAG Remote Bit Bang, instead of affecting one randomly.
remote_bitbang_t(port); instead of remote_bitbang_t(0);
If no port is selected default value is 0, then the port is chosen randomly.
Tested with OpenOCD + GDB

https://github.com/freechipsproject/rocket-chip/issues/1339

### CODE REVIEW

File: src/main/resources/csrc/emulator.cc [hunk 1]
```diff
@@ -69,6 +69,10 @@ EMULATOR OPTIONS\n\
   -m, --max-cycles=CYCLES  Kill the emulation after CYCLES\n\
        +max-cycles=CYCLES\n\
   -s, --seed=SEED          Use random number seed SEED\n\
+
+      --rbb-port=PORT      Use PORT for remote bit bang (with OpenOCD and GDB) \n\
```
Comment (mwachs5) [inline src/main/resources/csrc/emulator.cc]: Can you add the “short” version to the help here?

File: src/main/resources/csrc/emulator.cc [hunk 2]
```diff
@@ -110,6 +114,8 @@ int main(int argc, char** argv)
   uint64_t max_cycles = -1;
   int ret = 0;
   bool print_cycles = false;
+  // Port numbers are 16 bit unsigned integers. 
+  uint16_t port = 0;
```
Comment (mwachs5) [inline src/main/resources/csrc/emulator.cc]: Can you call this “rbb_port”

File: src/main/resources/csrc/emulator.cc (hunk 1 above)
Comment (noureddine-as) [reply src/main/resources/csrc/emulator.cc]: Done.

File: src/main/resources/csrc/emulator.cc (hunk 2 above)
Comment (noureddine-as) [reply src/main/resources/csrc/emulator.cc]: Done.

**Pull request merged**